<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminLogin</class>
 <widget class="QDialog" name="AdminLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1E1E1E;</string>
  </property>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>361</y>
     <width>256</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: none;
    border-bottom: 0px solid black;
    background-color: white;
    color: black;
    font-weight: bold;
    font-family: &quot;Roboto&quot;;
    font-size: 16px;
    padding-left: 10px;
}
</string>
   </property>
   <property name="placeholderText">
    <string>Enter Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>425</y>
     <width>256</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: none;
    border-bottom: 0px solid black;
    background-color: white;
    color: black;
    font-weight: bold;
    font-family: &quot;Roboto&quot;;
    font-size: 16px;
    padding-left: 10px;
}
</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Enter Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>509</y>
     <width>256</width>
     <height>36</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 7px;
    border: 0px solid black;
    background-color: rgb(0, 0, 255);
    color: white;
    font-size: 18px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: rgb(50, 50, 255);
}

QPushButton:pressed {
    background-color: rgb(0, 0, 200); 
}
</string>
   </property>
   <property name="text">
    <string>Signin</string>
   </property>
  </widget>
  <widget class="QLabel" name="invalid">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>460</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>428</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel, QPushButton {
    background-color: rgba(255, 255, 255, 0); 
border: 0px;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>312</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> color: white;
    font-weight: bold;
    font-family: &quot;Roboto&quot;;
    font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Admin login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>0</y>
     <width>445</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>136</y>
     <width>372</width>
     <height>174</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>email</zorder>
  <zorder>login</zorder>
  <zorder>label</zorder>
  <zorder>invalid</zorder>
  <zorder>password</zorder>
  <zorder>label_3</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
